test/clj/game/cards/programs_test.clj

```diff
@@ -3755,7 +3754,7 @@
         (click-prompt state :runner "Yes")
         (click-card state :runner (get-ice state :hq 1))
         (click-card state :runner (get-ice state :hq 0))
-        (is (= 1 (count (:hand (get-runner))))))))
+        (is (= 0 (count (:hand (get-runner))))))))
```

Comment:
Can there be a more explicit check in addition to this? Either a message or a check for a card name in a new place or something?

Reply:
How about this change?